### PR TITLE
include analytic module

### LINKED ISSUE #26: analytic submodule not included in pip package
Hi Everyone,

in the Demo Notebook `energy_decay_curves_and_reverberation_time` the line `from pyrato import analytic` throws an import error:
```python
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Cell In[3], line 1
----> 1 from pyrato import analytic

ImportError: cannot import name 'analytic' from 'pyrato' (/srv/conda/envs/notebook/lib/python3.10/site-packages/pyrato/__init__.py)
```
The `analytic` submodule does not seem to be included in the package distributed by pip. This happens locally on my machine, but also in the [binder notebook](https://notebooks.gesis.org/binder/jupyter/user/pyfar-pyrato-h7sz3kzk/notebooks/examples/energy_decay_curves_and_reverberation_time.ipynb).

Best,
Tobi

### PR DESCRIPTION
**Pull request by hoyer-a**
Include analytic module.

closes #26 

I had to make some changes to have the docs build succeed. 

Similar problems with auto-numbered footnotes as in https://github.com/pyfar/spharpy/pull/74. Seems like sphinx doesn't like it when auto-numbered footnotes appear twice (in the short function summary at the top and in the detailed function documentation).

### CODE REVIEW

**Review by ahms5 — CHANGES_REQUESTED** [review]
Thanks for taking care of it. It looks fine, I'm just wondering about the autonumbreing of the references.

File: pyrato/analytic/analytic.py [hunk 1]
```diff
@@ -5,7 +5,8 @@
 
 def eigenfrequencies_rectangular_room_rigid(
         dimensions, max_freq, speed_of_sound=343.9, sort=True):
-    """Calculate the eigenfrequencies of a rectangular room with rigid walls.
+    """Calculate the eigenfrequencies of a rectangular room with rigid
+    walls [1]_.
```
[the remark this answers is not in the document]
Comment (hoyer-a) [reply pyrato/analytic/analytic.py:9]: This produces the "Too many autonumbered footnote references" error. Would have to restructure the docstring a bit to make it work.

 Maybe a general revision of the pyrato docs in a separate PR would make sense.

**Review by ahms5 — APPROVED** [review]

File: pyrato/analytic/__init__.py [hunk 2]
```diff
@@ -3,7 +3,30 @@
...
+    transcendental_equation_eigenfrequencies_impedance,
+    transcendental_equation_eigenfrequencies_impedance_newton,
+    gradient_trancendental_equation_eigenfrequencies_impedance,
+    initial_solution_transcendental_equation,
+    eigenfrequencies_rectangular_room_1d,
+    normal_eigenfrequencies_rectangular_room_impedance,
+    eigenfrequencies_rectangular_room_impedance,
+    mode_function_impedance,
+    pressure_modal_superposition,
+    rectangular_room_impedance,
```
Comment (mberz) [inline pyrato/analytic/__init__.py:16]:
```suggestion
    eigenfrequencies_rectangular_room_impedance,
    rectangular_room_impedance,
```

Comment (mberz) [reply pyrato/analytic/__init__.py:16]: Most of these functions are intended for internal use only

File: pyrato/analytic/__init__.py [hunk 3]
```diff
@@ -3,7 +3,30 @@
...
+    'transcendental_equation_eigenfrequencies_impedance',
+    'transcendental_equation_eigenfrequencies_impedance_newton',
+    'gradient_trancendental_equation_eigenfrequencies_impedance',
+    'initial_solution_transcendental_equation',
+    'eigenfrequencies_rectangular_room_1d',
+    'normal_eigenfrequencies_rectangular_room_impedance',
+    'eigenfrequencies_rectangular_room_impedance',
+    'mode_function_impedance',
+    'pressure_modal_superposition',
+    'rectangular_room_impedance',
```
Comment (mberz) [inline pyrato/analytic/__init__.py:31]:
```suggestion
    'eigenfrequencies_rectangular_room_impedance',
    'rectangular_room_impedance',
```
Same as above

**Review by mberz — CHANGES_REQUESTED** [review]
I've marked a couple of functions to be removed from the init file, since they're intended for internal use only.
I guess this will reduce the problems with sphinx a bit as well.

**Review by mberz — APPROVED** [review]
Looks good, can be merged after #30 has been adapted and merged.

**Pull request merged**